api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -3802,7 +3812,7 @@
         "type": "object"
       },
       "io.k8s.api.core.v1.NodeRuntimeHandlerFeatures": {
-        "description": "NodeRuntimeHandlerFeatures is a set of runtime features.",
+        "description": "NodeRuntimeHandlerFeatures is a set of features implemented by the runtime handler. The features contained in the NodeRuntimeHandlerFeatures should depend on the runtime handlers, (i.e. dependent to the information exposed by the OCI runtime-spec's features).",
```

Comment:
```suggestion
        "description": "NodeRuntimeHandlerFeatures is a set of features implemented by the runtime handler. ",
```

Reply:
fixed in 
- K8s API: 5d75660dc11ff443ebab2551aed8e56a54cc218d
- CRI: 38e4c6b01666128216290e1579a8f2d408a60e66